### PR TITLE
Create Ballerina.toml by default from the init command

### PR DESCRIPTION
**Pull request by NatashaWso2**
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/9462

### CODE REVIEW

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java [hunk 1]
```diff
@@ -86,10 +86,11 @@ public void execute() {
             if (interactiveFlag) {
 
                 // Check if Ballerina.toml file needs to be created.
-                out.print("Create Ballerina.toml [yes/y, no/n]: (n) ");
+                out.print("Create Ballerina.toml [yes/y, no/n]: (y) ");
                 String createToml = scanner.nextLine().trim();
 
-                if (createToml.equalsIgnoreCase("yes") || createToml.equalsIgnoreCase("y")) {
+                if (createToml.equalsIgnoreCase("yes") || createToml.equalsIgnoreCase("y") ||
+                        createToml.isEmpty()) {
```
Comment (hemikak) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java]: Can we bring this to the same line ?

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java [hunk 2]
```diff
@@ -144,12 +140,13 @@ public void execute() {
...
                     }
-
-                    first = false;
                 } while (!validInput);
 
                 out.print("\n");
             } else {
+                manifest = new Manifest();
+                manifest.setName(guessOrgName());
+                manifest.setVersion(DEFAULT_VERSION);
```
Comment (hemikak) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java]: What would happen if theres already a Ballerina.toml file ?

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java [hunk 3]
```diff
@@ -111,17 +112,12 @@ public void execute() {
...
-                        out.print("Ballerina source [service/s, main/m]: (s) ");
-                    } else {
-                        out.print("Ballerina source [service/s, main/m, finish/f]: (f) ");
-                    }
+                    out.print("Ballerina source [service/s, main/m, finish/f]: (f) ");
                     srcInput = scanner.nextLine().trim();
 
                     if (srcInput.equalsIgnoreCase("service") || srcInput.equalsIgnoreCase("s") ||
-                            (first && srcInput.isEmpty())) {
+                            srcInput.isEmpty()) {
```
Comment (hemikak) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java]: Can we bring this to the same line ?

**Comment by hemikak** [comment]
If a user already has a existing Ballerina project, running "ballerina init" would create them a Ballerina.toml file ?

**Comment by hemikak** [comment]
Lets add a reference to the email discussion which prompted this change.

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java (hunk 1 above)
Comment (NatashaWso2) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java]: Fixed with f52251b

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java (hunk 2 above)
Comment (NatashaWso2) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java]: If there's already a Ballerina.toml inside the project, it is not created again

**Pull request merged**